python/cudf/cudf/core/internals/where.py

```diff
@@ -0,0 +1,375 @@
+import warnings
+
+import numpy as np
+import pandas as pd
+
+import cudf
+
+
+def _normalize_scalars(col, other):
+    """
+    Try to normalizes scalar values as per col dtype
+    """
+    if (
+        other is not None
```

Comment:
I think this `other is not None` check is redundant because we check `isinstance(other, float)` below